### PR TITLE
feat(mockServer): 实现官网landing模板

### PR DESCRIPTION
**Pull request by spongebody**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Background and solution

实现官网landing模板。
入口
image

UI效果
image
image
image
image
image

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Introduced a new component, "TinyNavMenu," for enhanced navigation.
	- Added a required "data" property to the TinyNavMenu for menu item configuration.
	- Included the TinyNavMenu in the content structure for easy integration.
	- Enhanced configuration options for the TinyNavMenu, including display overflow settings and context menu actions.

### CODE REVIEW

**Review by stwflyfox — APPROVED** [review]

File: packages/design-core/public/mock/bundle.json [hunk 1]
```diff
@@ -6005,6 +6005,91 @@
...
+                },
+                "collapse": {
+                  "number": 6,
+                  "text": {
+                    "zh_CN": "显示更多"
+                  }
+                },
+                "content": [
+                  {
+                    "property": "data",
```
Comment (hexqi) [inline packages/design-core/public/mock/bundle.json:6045]: 导航菜单只有一个data属性吗，可以把部分常用的属性也加进来

File: packages/design-core/public/mock/bundle.json [hunk 2]
```diff
@@ -6005,6 +6005,91 @@
...
+              "childWhitelist": "",
+              "parentWhitelist": "",
+              "descendantBlacklist": "",
+              "ancestorWhitelist": ""
+            },
+            "isNullNode": false,
+            "isLayout": false,
+            "rootSelector": "",
+            "shortcuts": {
+              "properties": ["text", "size"]
```
Comment (hexqi) [inline packages/design-core/public/mock/bundle.json]: 这里要改下，这里是定义在画布中快捷设置组件属性的

File: packages/design-core/public/mock/bundle.json (hunk 1 above)
Comment (spongebody) [reply packages/design-core/public/mock/bundle.json:6045]: 好的，已经添加了常用属性

File: packages/design-core/public/mock/bundle.json (hunk 2 above)
Comment (spongebody) [reply packages/design-core/public/mock/bundle.json]: 已修改

**Review by hexqi — APPROVED** [review]

**Pull request merged**